pages/en/lb4/Preparing-API-for-consumption.md

```diff
@@ -0,0 +1,50 @@
+---
+lang: en
+title: 'Preparing API for Consumption'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Preparing-API-for-consumption.html
+summary:
+---
+
+## Preparing your API for consumption
```

Comment:
nitpick: maybe add some note to remind people to follow previous steps.

e.g. In this LB3 tutorial: http://loopback.io/doc/en/lb3/Getting-started-with-LoopBack.html
There is a note on page "Use API Explorer": http://loopback.io/doc/en/lb3/Use-API-Explorer.html

>  Note: If you followed Create a simple API, keep that app running and skip down to Run API Explorer.
If you’re just jumping in, follow the steps below to catch up…